src/nvim/tui/term_input.inl

```diff
@@ -257,19 +257,7 @@ static TermInput *term_input_new(void)
   rv->paste_enabled = false;
   rv->in_fd = 0;
 
-  // Set terminal encoding based on environment(taken from libtermkey source
-  // code)
-  const char *e;
-  int flags = 0;
-  if (((e = os_getenv("LANG")) || (e = os_getenv("LC_MESSAGES"))
-        || (e = os_getenv("LC_ALL"))) && (e = strchr(e, '.')) && e++ &&
-      (strcasecmp(e, "UTF-8") == 0 || strcasecmp(e, "UTF8") == 0)) {
-    flags |= TERMKEY_FLAG_UTF8;
-  } else {
-    flags |= TERMKEY_FLAG_RAW;
-  }
-
-  rv->tk = termkey_new_abstract(os_getenv("TERM"), flags);
+  rv->tk = termkey_new_abstract(os_getenv("TERM"), 0);
```

Comment:
It might just be that you haven't rebased, but we can't send the result of `os_getenv()` directly to `termkey_new_abstract()`. See #2747